scripts/get-keptn-payload.js

```diff
@@ -0,0 +1,25 @@
+module.exports = ({ context }) => {
+  const labels = context.payload.pull_request.labels;
+  const parsedLabels = {};
+  labels.forEach((label) => {
+    const [key, value] = label.name.split(":");
+    parsedLabels[key] = value;
+  });
+
+  return JSON.stringify({
+    data: {
+      project: parsedLabels["project"],
+      stage: parsedLabels["stage"],
+      service: parsedLabels["service"],
+      // What should we send if the PR is closed without merging?
+      status: "succeeded",
+      result: "pass",
+    },
+    source: "github",
+    specversion: "1.0",
+    // TODO see what the event name is
```

Comment:
Not sure what the event name should be.

Reply:
The event must be: `sh.keptn.event.enable-feature.finished`